src/components/FilterButton/FilterButton.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './FilterButton.css';
+
+function FilterButton() {
+  return (
+    <button type="button">Skills</button>
+  );
+}
+
+export default FilterButton;
```

Comment:
This has no place in this commit, and no place in the PR either, this PR is only about the tile, filters will come in a different ticket. Remove these from this commit.